target/zephyr/Kconfig

```diff
@@ -421,6 +431,13 @@ if MENDER_MCU_CLIENT
                 help
                     A secondary security tag that ROOT CA server credential will be referenced with, typically used to authenticate the server from where to download Mender Artifacts. See tls_credential_add.
 
+            config MENDER_NET_CA_CERTIFICATE_TAG_DORMANT
+                int "Dormant CA certificate tag for Server rescue"
+                default 3
```

Comment:
For rotating credentials at some point, likely the user needs for certificates: old server + artifact store + new server + artifact store. Should we set then the TAG here to 9 or something like that? It is just an arbitrary number.

Alternatively we could have like a base number for MENDER_NET_CA_CERTIFICATE_TAG_PRIMARY, and the rest be +1, +2, etc :thinking:

Reply:
> Alternatively we could have like a base number for MENDER_NET_CA_CERTIFICATE_TAG_PRIMARY, and the rest be +1, +2, etc 🤔

I like this. It's similar to the `sector offset` I did in the partitioning changes. It gives a user an easy way to leave some room for other stuff if that's what they need.